CONTRIBUTING.md

```diff
@@ -37,7 +37,7 @@ If you run into any trouble, please reach out to us on the issue you are working
 
 In order to make changes to Gradle, you'll need:
 
-* A [Java Development Kit](http://jdk.java.net/) (JDK) **version 11**.
+* A [Java Development Kit](http://jdk.java.net/) (JDK) **version 11**. Fixed version is required to use remote cache. Read more on that [here](https://github.com/gradle/gradle-org-conventions-plugin#what-it-does).
```

Comment:
This might be more appropriate under "Useful tips" below.  This was written to be just a bullet list of the minimums of what you need without explanation about why you need them. This is to present low friction for people getting started.

We could have a section below like:
```
## Useful tips
...
### Slow local builds

Gradle, Inc runs a set of remote build cache nodes to speed up local builds when developing Gradle. By default, the build is [configured](https://github.com/gradle/gradle-org-conventions-plugin#what-it-does) to use the build cache node in the EU region.

The build cache has anonymous read access, so you don't need to authenticate in order to use it. You can use a different build cache node by specifying -DcacheNode=us for a build cache node in the US or -DcacheNode=au for a build cache node in Australia.

If you are not getting cache hits from the build cache, you may be using the wrong version of Java. A fixed version (Java 11) is required to get remote cache hits.
```

Reply:
Thanks a lot! 
I think that it is still useful to mention the reason behind fixed Java version in the bullet point to avoid confusion for people who haven't read the whole document yet. Wdyt?